config/jobs/kubernetes-security/generated-security-jobs.yaml

```diff
@@ -506,8 +503,7 @@ presubmits:
         command:
         - runner.sh
         - kubetest
-        image: gcr.io/k8s-testimages/kubekins-e2e:latest-master
-        imagePullPolicy: Always
+        image: gcr.io/k8s-testimages/kubekins-e2e:v20190628-9455316-master
```

Comment:
Why are we switching this tag and pull policy?

Reply:
these are debug left overs.

the correct way is to use the appropriate kubekins:
- tagged for a k8s branch
- not always pre-pulling the latest one